### PR TITLE
MESH-1439: send X-smartstack-expected-service header to hacheck, cleanup

### PR DESCRIPTION
**Pull request by BeryJu**
this also fixes the build on bionic, and removes leftover bintray and travis things

### CODE REVIEW

File: src/nerve_tools/configure_nerve.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def generate_subconfiguration(
     advertise = service_info.get('advertise', ['region'])
     extra_advertise = service_info.get('extra_advertise', [])
     healthcheck_headers = service_info.get('extra_healthcheck_headers', {})
+    healthcheck_headers["X-smartstack-expected-service"] = service_name
```
Comment (Buffer0x7cd) [inline src/nerve_tools/configure_nerve.py:129]: nit: use small case with headers name ( although it's just synthetic suger)

**Comment by Buffer0x7cd** [comment]
Can you also attach the generated nerve config ?

**Comment by BeryJu** [comment]
> Can you also attach the generated nerve config ?
```
        "yelp-main.main_ro.pnw-devc:10.93.9.107.8888.v2.new": {
            "check_interval": 3.0,
            "checks": [
                {
                    "fall": 2,
                    "headers": {
                        "Host": "www.yelp.com",
                        "X-Proxiedssl": "yes",
                        "X-yelp-expected-service": "yelp-main.main_ro"
                    },
                    "host": "10.93.9.107",
                    "open_timeout": 2,
                    "port": 6666,
                    "rise": 1,
                    "timeout": 2,
                    "type": "http",
                    "uri": "/http/yelp-main.main_ro/8888/status"
                }
            ],
            "host": "10.93.9.107",
            "labels": {
                "habitat:uswest2adevc": "",
                "habitat:uswest2bdevc": "",
                "habitat:uswest2cdevc": "",
                "region:uswest2-devc": ""
            },
            "port": 8888,
            "weight": 10,
            "zk_hosts": [
                "10.81.16.240:22181",
                "10.81.22.142:22181",
                "10.81.25.26:22181"
            ],
            "zk_path": "/smartstack/global/yelp-main.main_ro"
        }
```

**Review by ankit864 — APPROVED** [review]
LGTM!

**Pull request merged**